website_product_filters/models/product.py

```diff
@@ -80,11 +80,10 @@ class ProductCategory(models.Model):
         "product_template_id", readonly=True)
     total_tree_products = fields.Integer("Total Subcategory Prods",
                                          compute="_get_product_count",
-                                         store=True,)
+                                         store=False,)
```

Comment:
@oscarolar I prefere store = True with a good trigger, but I do not know if it is technically possible, compute this field any time view is loaded is too time consuming.

Can you propose a solution for that?